db/schema.yml

```diff
@@ -681,14 +681,14 @@ container_groups:
 - dns_policy
 - ems_id
 - container_node_id
-- last_perf_capture_on
 - container_replicator_id
 - ipaddress
 - type
 - container_project_id
 - phase
 - message
 - reason
+- last_perf_capture_on
```

Comment:
That command just dumps your schema to the file. So if you had more changes than just your migration it means *your* schema is wrong. I would suggest using that rake task in conjunction with `rake evm:db:reset` to get the correct ordering. I usually do `RAILS_ENV=test bundle exec rake evm:db:reset evm:db:write_schema`